contrib/collectd/template.py

```diff
@@ -0,0 +1,52 @@
+pkgname = "collectd"
+pkgver = "5.12.0"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+        "--prefix=/usr/bin",
```

Comment:
this is just wrong, since then it installs e.g. /usr/bin/lib /usr/bin/include etc. the default =/usr is correct

Reply:
changing it to /usr makes your install of contrib to /usr/share/collectd conflict, but you can put it somewhere else or just not install it at all (it seems to be a collection of random files, most of which are useless)

Reply:
> it seems to be a collection of random files, most of which are useless
they're referenced in the man pages, and all of those files are example configurations for various plugins so I'm making an effort to include them.  putting them in /usr/share/doc (where they're documented to be in the man page) seems fine, since it's what the other doc packages do as well.  lmk if it isn't, though.